lib/dialects/mssql/query-generator.js

```diff
@@ -181,13 +181,18 @@ var QueryGenerator = {
     var query = 'ALTER TABLE <%= tableName %> ALTER COLUMN <%= attributes %>;';
     var attrString = [];
 
-    for (var attrName in attributes) {
-      var definition = attributes[attrName];
-
-      attrString.push(Utils._.template('<%= attrName %> <%= definition %>')({
-        attrName: this.quoteIdentifier(attrName),
-        definition: definition
-      }));
+    for (var attributeName in attributes) {
+      var definition = attributes[attributeName];
+      if (definition.match(/REFERENCES/)) {
+        query = query.replace('ALTER COLUMN', '');
+        definition = definition.replace(/.+?(?=REFERENCES)/,'');
+        attrString.push('ADD CONSTRAINT ' + this.quoteIdentifier(attributeName + '_foreign_idx') + ' FOREIGN KEY (' + this.quoteIdentifier(attributeName) + ') ' + definition);
+      } else {
+        attrString.push(Utils._.template('<%= attrName %> <%= definition %>')({
+          attrName: this.quoteIdentifier(attributeName),
+          definition: definition
+        }));
+      }
```

Comment:
Right but it's still matching REFERENCES against the SQL.
This syntax will not support both changing a columns default value and setting a constraint will it?